diff --git setup.py setup.py
index 5766987..a4b2f9c 100644
--- setup.py
+++ setup.py
@@ -43,7 +43,7 @@ if __name__ == "__main__":
           namespace_packages = ['scikits'],
           packages = find_packages(),
           package_data = {
-              "": ["*.mtx.gz"],
+              "": ["test_data/*.mtx.gz"],
               },
           test_suite="nose.collector",
           # Well, technically zipping the package will work, but since it's
